### PR TITLE
Ja task124 final

### PR DESCRIPTION
**Pull request by JamesAttwood1910**
Hey Marcus PR that reslves resolves KlimaDAO/dash-apps/#124

I've pushed my lintr changes. 

Let me know what you think, and if any other changes are needed.

### CODE REVIEW

File: src/apps/tco2_dashboard/helpers.py [hunk 1]
```diff
@@ -698,3 +698,158 @@ def uni_v2_pool_price(web3, pool_address, decimals, base_price=1):
...
+    retirements_all['year'] = (
+        retirements_all["dailyKlimaRetirements_datetime"].dt.year
+    )
+    retirements_all['year'] = retirements_all['year'].apply(str)
+    retirements_all['month_year'] = (
+        retirements_all.month.str.cat(retirements_all.year, sep='-')
+    )
+    retirements_all = retirements_all.drop(
+        ['year', 'month', 'dailyKlimaRetirements_datetime'], axis=1
+        )
```
Comment (0xAurelius) [inline src/apps/tco2_dashboard/helpers.py:719]: Minor whitespace issue here: one extra tab

Comment (JamesAttwood1910) [reply src/apps/tco2_dashboard/helpers.py]: Delted

**Comment by 0xAurelius** [comment]
Great work on this!

Some minor feedback:
1. Please check the font styling on the table data and headers - it should match the styles applied to the table below
2. `tCO2e` is a nice technical designation but since we're already talking about carbon credits which abstract away the different emissions factors, we can just say "Tonnes"
3. A shorter date format will make this more easy to view on smaller format device: MMM-YY (e.g. Jan-23) instead of MMMM-YYYY (e.g. January 2023)

**Comment by JamesAttwood1910** [comment]
Thanks! I'll get those changes made today

**Comment by JamesAttwood1910** [comment]
Hey @0xAurelius I've made the changes.

**Review by 0xAurelius — APPROVED** [review]
LGTM 
image

**Pull request merged**